packages/datetime/src/common/dateUtils.ts

```diff
@@ -110,6 +110,27 @@ export function isMonthInRange(date: Date, dateRange: DateRange) {
     return start <= day && day <= end;
 }
 
+export function isTimeInRange(time: Date, minTime: Date, maxTime: Date): boolean {
+    time = getDateOnlyWithTime(time);
+    minTime = getDateOnlyWithTime(minTime);
+    maxTime = getDateOnlyWithTime(maxTime);
+
+    const isTimeAfterMinTime = time.getTime() >= minTime.getTime();
+    const isTimeBeforeMaxTime = time.getTime() <= maxTime.getTime();
+
+    if (maxTime.getTime() <= minTime.getTime()) {
```

Comment:
Well, it's not state error, it's a special case when we have time range that past midnight. For example `minTime=22:00` to `maxTime=2:00` then `minTime` > `maxTime`. 

If I select time after midnight e.g. `1:00` then `isTimeAfterMinTime` becomes `false` and `isTimeBeforeMaxTime` becomes `true`.

If I select time before midnight e.g. `23:00` then `isTimeAfterMinTime` becomes `true` and `isTimeBeforeMaxTime` becomes `false`.

So if selected time is in a range that past midnight, one of this must be `true`.

I think variable names are a bit misleading. I think I should name them `isTimeGreaterThanMinTime` and `isTimeSmallerThanMaxTime` are more accurate? What do you think? :)